sklearn/ensemble/tests/test_gradient_boosting_loss_functions.py

```diff
@@ -311,3 +313,6 @@ def test_lad_equals_quantile_50(seed):
     lad_weighted_loss = lad(y_true, raw_predictions, sample_weight=weights)
     ql_weighted_loss = ql(y_true, raw_predictions, sample_weight=weights)
     assert lad_weighted_loss == approx(2 * ql_weighted_loss)
+    pbl_weighted_loss = pinball_loss(y_true, raw_predictions,
+                                     sample_weight=weights)
+    assert pbl_weighted_loss == approx(ql_weighted_loss)
```

Comment:
Could you please add a similar consistency test for weighted quantile with `alpha != 0.5`.

Reply:
done

Reply:
Thanks.